data-service/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration packages="com.amazonaws.services.lambda.runtime.log4j2.LambdaAppender">
```

Comment:
If you rebase this on the latest `dev` commit, you can omit this configuration (it's included in the `lambda-logging` project).